tests/ping6_clicker.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2017
+#
+# All rights reserved.
+#
+# This file is distributed under the Clear BSD license.
+# The full text can be found in LICENSE in the root directory.
+
+import rootfs_boot
+from devices import board
+
+class BoardPing6ClickerDev(rootfs_boot.RootFSBootTest):
+	'''Board lowpan interface can ping6 the clicker.'''
+	def runTest(self):
+	# Make Lan-device ping Wan-Device
+		board.sendline('\nping6 -I lowpan0 -c 20 fe80:0000:0000:0000:0204:a30b:0000:c184')
```

Comment:
I think somewhere in the test doc we should mention the distance between the DUT and clicker device, which should be maintain constant throughout.